pkg/table/table.go

```diff
@@ -233,6 +252,14 @@ func (m *Model) StopSpinner() {
 func (m *Model) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
 	switch msg := msg.(type) {
 	case tea.KeyMsg:
+		if m.filterInputEnabled && !key.Matches(msg, m.KeyMap.ForceQuit, m.KeyMap.Filter) {
+			i, cmd := m.filterInput.Update(msg)
+			m.filterInput = i
+			m.updateFilteredRows()
+			m.updatePagination()
+			m.updateColumnWidths()
+			return m, cmd
+		}
 		switch {
```

Comment:
This `if` statement can be moved into the `switch` statement

```suggestion
		switch {
		case m.filterInputEnabled && !key.Matches(msg, m.KeyMap.ForceQuit, m.KeyMap.Filter):
			i, cmd := m.filterInput.Update(msg)
			m.filterInput = i
			m.updateFilteredRows()
			m.updatePagination()
			m.updateColumnWidths()
			return m, cmd
```